team.html

```diff
@@ -189,6 +141,101 @@ <h4 class="card-heading">Daniel<br>Feismann</h4>
+              <p id="w-node-_20a763cb-c617-37db-f11c-9216c96b1efb-98e4b81b" class="md">Ph.D. Electrical Engineering</p><img src="images/research.svg" loading="lazy" alt="">
+              <p id="w-node-c510cc05-bc66-3ad9-fc76-9fc8a9a7c91f-98e4b81b" class="md">Agent-based Modeling and Simulation of Power Systems</p><img src="images/at.svg" loading="lazy" id="w-node-_3f622320-565e-b878-6c55-7c6cd9778341-98e4b81b" alt="">
+              <div id="w-node-_3b540832-177e-db0f-332d-e324adc4b15c-98e4b81b">
+                <a id="w-node-_3b540832-177e-db0f-332d-e324adc4b15d-98e4b81b" href="mailto:johannes.hiry@tu-dortmund.de" class="contact-link profile">Send an e-mail</a>
+              </div>
+            </div>
+            <div class="card-footer">
+              <div class="h-flex full-width socials">
+                <a href="https://github.com/johanneshiry" class="socials-link w-inline-block"><img src="images/github.svg" loading="lazy" alt=""></a>
+                <a href="https://www.linkedin.com/in/johannes-hiry/" class="socials-link w-inline-block"><img src="images/linkedin.svg" loading="lazy" alt=""></a>
+                <a href="https://orcid.org/0000-0002-1447-0607" target="_blank" class="socials-link w-inline-block"><img src="images/orcird.svg" loading="lazy" alt="" class="image-21"></a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df1e-98e4b81b" class="card">
+          <div class="card-content">
+            <div class="h-flex"><img src="images/chris.png" loading="lazy" alt="">
+              <div class="full-width">
+                <h4 class="card-heading">Chris<br>Kittl</h4>
+              </div>
+            </div>
+            <div class="card-content-block"><img src="images/work.svg" loading="lazy" id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df27-98e4b81b" alt="">
+              <p id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df28-98e4b81b" class="md">Former Research Associate</p><img src="images/graduation-hat.svg" loading="lazy" id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df2a-98e4b81b" alt="">
+              <p id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df2b-98e4b81b" class="md">Ph.D. Electrical Engineering</p><img src="images/research.svg" loading="lazy" alt="">
+              <p id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df2e-98e4b81b" class="md">Efficient Large Scale Power System Simulations</p><img src="images/at.svg" loading="lazy" id="w-node-ce3c068c-09b1-ae2c-4872-7d7f7097df30-98e4b81b" alt="">
+              <div id="w-node-_0706eae7-f7ea-613c-1b7c-c9ef90ed48d1-98e4b81b">
+                <a id="w-node-_0706eae7-f7ea-613c-1b7c-c9ef90ed48d2-98e4b81b" href="mailto:chris.kittl@tu-dortmund.de" class="contact-link profile">Send an e-mail</a>
```

Comment:
I don't know if they're still checking their uni e-mail. Maybe we should remove these links for both of them?

Reply:
I would leave them in. For a general contact to Simona-team we have Thomas as contact person and here one could make an PR to have changes if they don't use it any longer or don't want to have it published here.